acqui/invoice.pl

```diff
@@ -108,14 +109,55 @@ =head1 DESCRIPTION
         exit 0;
     }
 }
+elsif ( $op && $op eq 'del_adj' ) {
```

Comment:
Hmm, I'd have likely tried to be a bit more modern here and pushed this into an API route and used a vue.js (or whatever we're aiming to use in Koha) component rather than full template reloads.